packages/block-library/src/file/edit.js

```diff
@@ -136,7 +142,7 @@ class FileEdit extends Component {
 		if ( ! href || hasError ) {
 			return (
 				<MediaPlaceholder
-					icon="media-default"
+					icon=<BlockIcon icon={ icon } />
```

Comment:
Can it follow other definitions of props?:
```suggestion
					icon={ <BlockIcon icon={ icon } /> }
```